lang/cs/Org.Apache.REEF.Common/Files/REEFFileNames.cs

```diff
@@ -50,6 +50,8 @@ public sealed class REEFFileNames
         private const string CLR_DRIVER_CONFIGURATION_NAME = "clrdriver.conf";
         private const string BRIDGE_DLL_NAME = "Org.Apache.REEF.Bridge.dll";
 
+        public const string CLR_BRIDGE_CONFIGURATION_NAME = "clrBridge.config";
```

Comment:
Let me remove the changes related to `REEFFileNames` from this PR and move them to separate JIRA, so that this PR can focus on APIs which are actually removed. Then we'll handle this constant separately.